release/changes.sh

```diff
@@ -91,12 +96,23 @@ mapfile -t exclude < <(printf "%s\n" "${sections[@]}" |
   cut -f2 -d, | sed -e 's/^/:(exclude)/')
 sections+=("Common Libraries,google/cloud,${exclude[*]}")
 
+# Use KMS as an exemplar. Assume that any changes made to it affect all
+# generated libraries, and thus should be grouped under "Common Libraries".
+mapfile -t common_messages < <(list_changes "${last_tag}" "google/cloud/kms" | grep -oE "\(#[0-9]+\)$")
+common_filter="$(printf "|%s$" "${common_messages[@]}")"
+common_filter="${common_filter:1}"
+common_filter="${common_filter//\(/\\\(}"
+common_filter="${common_filter//\)/\\\)}"
```

Comment:
Opt:  I might consider using `sed` instead of those built-ins.
```
common_filter=$(printf '|%s$' "${common_messages[@]}" |\
                sed -e 's/^|//' -e 's/[()]/\\&/g')
```
I think I find that clearer.

Reply:
Done. `&` must mean the matched string.

Reply:
Yes.  In particular, `sed(1)` says, _"The replacement may contain the special character & to refer to that portion of the pattern space which matched."_